servicex/transformer_manager.py

```diff
@@ -169,8 +198,18 @@ def launch_transformer_jobs(self, image, request_id, workers, chunk_size,
 
         self._create_job(api_v1, job, namespace)
 
+        autoscaler_api = kubernetes.client.AutoscalingV1Api()
+        hpa = self.create_hpa_object(request_id, workers)
+        self._create_hpa(autoscaler_api, hpa, namespace)
+
     @staticmethod
     def shutdown_transformer_job(request_id, namespace):
+        autoscaler_api = kubernetes.client.AutoscalingV1Api()
```

Comment:
Should there be a list of assets to be deleted instead of explicit code

Reply:
Ok - I'll assume this is "taken care of". :-)